addon/mixins/form-field-options.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+import Mixin from 'ember-metal/mixin';
+import get   from 'ember-metal/get';
+
+const { defineProperty } = Ember;
+
+export default Mixin.create({
+  didReceiveAttrs() {
+    this._super(...arguments);
+
+    let options = get(this, 'options') || {};
+
+    Object.keys(options).forEach((attributeName) => {
+      defineProperty(this, attributeName, { value: get(options, attributeName) });
```

Comment:
We should take a look at make it a `reads`
(for now reads doesn't work for fields like keydown)

Try a computed, instead of reads